backend/routes.py

```diff
@@ -1,30 +1,42 @@
 from flask import Flask, jsonify, request
 import os
+from chatbot import chatbot
+import json
 from flask_cors import CORS
 
 app = Flask(__name__)
 CORS(app)
 
 #dummy data
-sentences = [{'sass' : 'sample_response_1'}, {'sass' : 'sample_response_2'}, {'sass' : 'sample_response_3'}]
+dummy = [{'sass' : 'sample_response_1'}, {'sass' : 'sample_response_2'}, {'sass' : 'sample_response_3'}]
 
-#main route defaults
+#main route (Landing Page)
 @app.route('/', methods = ['GET'])
 def main_route():
-        return jsonify({"about":"Default"})
+        return jsonify({"about" : "Default"})
 
-#endpoint GET ALL (UNUSED)
-@app.route('/sentences', methods = ['GET'])
-def return_All():
-    return jsonify({'sentences' : sentences})
+#endpoint that returns dummy data above (Used For Testing)
+@app.route('/dummy', methods = ['GET'])
+def return_dummy():
+    return jsonify({'dummy' : dummy})
             
-#endpoint POST
-@app.route('/sentences', methods = ['GET','POST'])
-def test_route():
-    sentence = {'sass' : request.json['sass']}
-
-    #sentences.append(sentence)
-    return jsonify({'sentences' : sentence})
+#endpoint that echos input (Used For Testing)
+@app.route('/repeat', methods = ['GET','POST'])
+def repeat_route():
+        if request.method == 'POST':
```

Comment:
Just make this a post so you don't need the if statement?

Reply:
might need get for testing to see if the endpoint is fine